library/core/src/default.rs

```diff
@@ -163,7 +161,6 @@ macro_rules! default_impl {
 default_impl! { (), (), "Returns the default value of `()`" }
 default_impl! { bool, false, "Returns the default value of `false`" }
 default_impl! { char, '\x00', "Returns the default value of `\\x00`" }
-default_impl! { AsciiChar, AsciiChar::Null, "Returns the default value of `Null`" }
```

Comment:
The prompt information of the std is modified here. Should this be within the scope of this PR?